src/python/pants/base/deprecated.py

```diff
@@ -300,7 +300,7 @@ def resolve_conflicting_options(
 ):
     """Utility for resolving an option that's been migrated to a new location.
 
-    This ensures that the user does not try to specify both options.
```

Comment:
Sure, and I'll improve it to explain more about how it works. I removed it because it was weird to explain one little detail, but not explain the rest of the _how_.

> This will check if either option was explicitly configured, and if so, use that. If both were configured, it will error. Otherwise, it will use the default value for the new, preferred option.